lib/index.js

```diff
@@ -236,6 +236,38 @@ exports.parseDublinCore = BBPromise.method(function(chtml){
 });
 
 /**
+ * Scrapes EPrints data given Cheerio loaded html object
+ * @param  {Object}   chtml html Cheerio object
+ * @return {Object}         BBPromise for EPrints metadata
+ */
+exports.parseEprints = BBPromise.method(function(chtml){
+
+	return exports.parseBase(
+		chtml,
+		['meta'],
+		'No EPrints metadata found in page',
+		function(element) {
+			var nameAttr = element.attr('name');
+
+			// If the element isn't an EPrints property, skip it
+			if (!nameAttr || nameAttr.substring(0, 8).toLowerCase() !== 'eprints.') {
+				return;
+			}
+
+			var property = nameAttr.substring(nameAttr.lastIndexOf('.') + 1);
+
+			// Lowercase the first character
+			property = property.charAt(0).toLowerCase() + property.substr(1);
```

Comment:
I sort of just copied from the DC implementation (if you didn't guess). It probably makes sense to do the whole thing.